### ISSUE TITLE
Change just a specific option

### ISSUE THREAD

**Issue by diegobill** [nicolaskruchten/pivottable#1100]
Hi!

Is it possible to change one specific option of the pivotUI without having to overwrite all the existing options?

Thanks

**Comment by EricRossC** [comment]
I think you can't. Even when you need to redraw the UI to add new data you have to recreate it, as far as I know.

**Comment by nicolaskruchten** [comment]
That's correct, it's all or nothing: the whole thing is rerendered any time any option is changed anyways.
